testkit/javadsl/src/main/scala/com/lightbend/lagom/javadsl/testkit/ServiceTest.scala

```diff
@@ -245,41 +301,65 @@ object ServiceTest {
     val now = DateTimeFormatter.ofPattern("yyMMddHHmmssSSS").format(LocalDateTime.now())
     val testName = s"ServiceTest_$now"
 
-    val b1 = new GuiceApplicationBuilder()
+    val initialBuilder = new GuiceApplicationBuilder()
       .bindings(sBind[TestServiceLocatorPort].to(testServiceLocatorPort))
       .bindings(sBind[ServiceLocator].to(classOf[TestServiceLocator]))
       .bindings(sBind[TopicFactory].to(classOf[TestTopicFactory]))
       .configure("play.akka.actor-system", testName)
 
     val log = Logger(getClass)
 
-    val b3 =
+    val finalBuilder =
       if (setup.cassandra) {
+
         val cassandraPort = CassandraLauncher.randomPort
         val cassandraDirectory = Files.createTempDirectory(testName).toFile
         val t0 = System.nanoTime()
-        CassandraLauncher.start(cassandraDirectory, LagomTestConfigResource, clean = false, port = cassandraPort,
-          CassandraLauncher.classpathForResources(LagomTestConfigResource))
+
+        CassandraLauncher.start(
+          cassandraDirectory,
+          LagomTestConfigResource,
+          clean = false,
+          port = cassandraPort,
+          CassandraLauncher.classpathForResources(LagomTestConfigResource)
+        )
+
         log.debug(s"Cassandra started in ${TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - t0)} ms")
-        val b2 = b1.configure(CassandraTestConfig.persistenceConfig(testName, cassandraPort))
+
+        initialBuilder
+          .configure(CassandraTestConfig.persistenceConfig(testName, cassandraPort))
           .configure("lagom.cluster.join-self", "on")
-        disableModules(b2, KafkaClientModule, KafkaBrokerModule)
+          .disableModules(KafkaClientModule, KafkaBrokerModule)
+
+      } else if (setup.jdbc) {
+
+        initialBuilder
+          .configure(CassandraTestConfig.clusterConfig())
```

Comment:
I guess we should move it somewhere other than `CassandraTestConfig` (doesn't have to be in this PR).

Reply:
Let's do it on this PR. Otherwise, that will be another open issue filling the backlog

Reply:
After a further look, it seems that this whole `CassandraTestConfig` needs a re-arrangement. 

Java and Scala modules are not aligned. Java has `CassandraTestConfig` and `TestUtil` while the Scala one has two variants of `TestUtil`. 

`CassandraTestConfig` (in java dsl) is also confusing, because it's not in `persistence-cassandra` module, but in `persistence`. I won't refactor it in this PR and send another one for that. Seems too intrusive and doesn't relates to this one. Is more about re-arrangement and clean-up.